### PR TITLE
update v3relay article from stub to article

### PR DESCRIPTION
**Pull request by paulkania**
Currently fails -> [text][topic version 3 transaction relay] doesn't want to formulate into html for some reason.

I hope the content of update is something the reviewers are happy with. Not a huge change, just added more details from what I could understand from the Sept/Oct 2022 bdev mailing lists.

### CODE REVIEW

**Comment by paulkania** [comment]
re. failed test -> Logs say the 'excerpt' header is missing. I assumed excerpts were only for stub articles, which this isn't anymore (with the added word-count).
will try to fix and re-check

**Comment by paulkania** [comment]
in case anyone is interested - the current test failure comes from the following:
I added hyperlinks via [this is text][topic version 3 transaction relay] manually. It didn't convert automatically to html.
I am at something of an impasse, since all other markdown files that use the [text][topic x] format seem to just work on their own.

File: _topics/en/version-3-transaction-relay.md [hunk 1]
```diff
@@ -46,20 +53,19 @@ see_also:
...
+non-standard rules designed to allow transaction replacement
+while precluding transaction-pinning attacks. V3 transactions also
+require minor changes to the Package RBF policy in order to maintain
+incentive compatibility with miners.
 
-## Optional.  Force the display (true) or non-display (false) of stub
-## topic notice.  Default is to display if the page.content is below a
-## threshold word count
-#stub: false
+V3 transaction relay solves rule 3 [transaction pinning][topic version 3 transaction relay]
```
Comment (harding) [inline _topics/en/version-3-transaction-relay.md:65]: I think this should link to `[topic transaction pinning]`

File: _topics/en/version-3-transaction-relay.md [hunk 2]
```diff
@@ -46,20 +53,19 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
+(e.g. minimum and max transaction weight) while also adding some
+non-standard rules designed to allow transaction replacement
```
Comment (harding) [inline _topics/en/version-3-transaction-relay.md:60]:
```suggestion
additional rules designed to allow [transaction replacement][topic rbf]
```

File: _topics/en/version-3-transaction-relay.md [hunk 3]
```diff
@@ -46,20 +53,19 @@ see_also:
...
+while precluding transaction-pinning attacks. V3 transactions also
+require minor changes to the Package RBF policy in order to maintain
+incentive compatibility with miners.
 
-## Optional.  Force the display (true) or non-display (false) of stub
-## topic notice.  Default is to display if the page.content is below a
-## threshold word count
-#stub: false
+V3 transaction relay solves rule 3 [transaction pinning][topic version 3 transaction relay]
+and may allow the removal of the CPFP carve-out.
```
Comment (harding) [inline _topics/en/version-3-transaction-relay.md:66]: CPFP carve out can link to `[topic cpfp carve out]`

File: _topics/en/version-3-transaction-relay.md [hunk 4]
```diff
@@ -46,20 +53,19 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
+(e.g. minimum and max transaction weight) while also adding some
+non-standard rules designed to allow transaction replacement
+while precluding transaction-pinning attacks. V3 transactions also
+require minor changes to the Package RBF policy in order to maintain
```
Comment (harding) [inline _topics/en/version-3-transaction-relay.md:62]: Package RBF can link to `[topic package relay]`.

**Review by harding — COMMENTED** [review]
Looks good to me with some suggested edits.  Thanks!

To answer your question, the reason `[topic version 3 transaction relay]` doesn't work is that this file defines a `shortname` of `v3 transaction relay`, making the reference link `[topic v3 transaction relay]`.

File: _topics/en/version-3-transaction-relay.md (hunk 3 above)
Comment (paulkania) [reply _topics/en/version-3-transaction-relay.md:66]: done :)

File: _topics/en/version-3-transaction-relay.md (hunk 4 above)
Comment (paulkania) [reply _topics/en/version-3-transaction-relay.md]: done :)

**Review by Xekyo — COMMENTED** [review]
LGTM, should perhaps be squashed

**Comment by paulkania** [comment]
@Xekyo 
1 I squashed the commits, accidentally to a new PR. Excuse me. [[#1004](https://github.com/bitcoinops/bitcoinops.github.io/pull/1004)]. As such I'll close this branch.
2. I noticed a link wasn't forming properly on my local build: [line 66] - topic ephemeral anchors. There is no shorthand for that topic, and ephemeral anchors is spelt correctly

**Pull request closed without merging**